Makefile

```diff
@@ -15,7 +15,7 @@ export PYTHON
 SPLIT_REGEX := ^\([^.]*\)\.\?\(.*\)$$
 
 # The platform to run on. It is made up of FPGA_MAIN_BOARD.EXPANSION_BOARD
-DEFAULT_PLATFORM = opsis
+DEFAULT_PLATFORM = tinyfpga_bx
```

Comment:
I think the icebreaker is a better default platform given that we don't have a usb serial UART yet.

Reply:
See rationale in https://github.com/timvideos/litex-buildenv/issues/134 :-)  Ie, almost no one has an Icebreaker yet, including me (I have two on order, including an early bird one; neither has shipped yet), but lots of people, including the person whose problem prompted deciding to change this now, have a TinyFPGA BX.  (And I have a TinyFPGA BX, so I can test it :-) )

I agree the `icebreaker` is a better final target.  I'll add a `FIXME` comment to that effect so we don't forget.

Ewen

Reply:
If we wait long enough, my first icebreaker should arrive (in theory the last of the earlybird orders ship this month, so I guess mine should arrive in the next 2-3 weeks... :-) ).  At which point I could test defaulting to the icebreaker and do that instead.  (I think we possibly still want some of the other changes in this pull request, to have useful TinyFPGA BX defaults.)

Ewen

Reply:
FTR, I've just learned that my iCEBreaker early bird order has been held back "to ship with the main production batch" (apparently it was a mistake to order a second one from the main batch, even though I paid separate shipping for each one :-( ).  

So "waiting long enough" probably means another couple of months.

Ewen